CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Overall**
+
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 45.28.0 to 48.2.3
+  * [UPGRADE] - Prometheus has been upgraded from version 2.44.0 to 2.45.0
+  * [UPGRADE] - Prometheus Operator has been upgraded from version 0.65.1 to 0.66.0
+  * [UPGRADE] - Grafana has been upgraded from version 9.5.5 to 10.0.3
+  * [UPGRADE] - Kube State Metrics has been upgraded from version 2.8.2 to 2.9.2
+  * [UPGRADE] - K8s-sidecar used with Grafana has been upgraded from version 1.5.0 to 1.6.0
```

Comment:
In my test environment, after fresh install, k8s-sidecar containers were using image: quay.io/kiwigrid/k8s-sidecar:1.24.6

Reply:
Most likely an artifact of not picking up the component_versions.env file.  I suggest taking care of that first and see whether this goes away.

Reply:
I got versions for the K8s-sidecar and the Node Exporter mixed up.  They'll be corrected in the next push.